HISTORY.rst

```diff
@@ -5,6 +5,10 @@ History
 Unreleased
 ----------
 
+* New glinter warnings were added (see bug 1682980):
+  - `SHORT_NAME`: When a category name, metric name or ping name is less than 3 characters.
+  - `SHORT_LABEL`: When a label is less than 2 characters.
+  - `SHORT_REASON`: When a ping reason is less than 2 characters.
```

Comment:
Grammar nit: append `long` to these, or rewrite to `has fewer than 2 characters`